jme3-dark-laf/src/org/jme3/netbeans/plaf/darkmonkey/DarkMonkeyLookAndFeel.java

```diff
@@ -29,9 +29,10 @@
 public class DarkMonkeyLookAndFeel extends com.nilo.plaf.nimrod.NimRODLookAndFeel{
     
     public static final String dmLAFDefault = "DarkMonkey.theme";
+    public static final String DEJA_VU_SANS_CONDENSED_PLAIN_12 = "DejaVu Sans Condensed-PLAIN-12";
+    public static final Font DEJA_VU_SANS_FONT = Font.decode(DEJA_VU_SANS_CONDENSED_PLAIN_12);
```

Comment:
I don't know how expensive decoding a Font is, but if we don't need it (on CJK), we could move the Decoding into `applyDejaVuFontIfSupported`

Reply:
Done